sites/docs/pages/getting-started.md

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 3
+hide_table_of_contents: false
+title: Build your first dashboard
+description: A 10 minute guide to building your first dashboard in Evidence.
+---
+<div id="objectives" class="alert svelte-17118v7">
+<strong>By the end of this 10 minute guide, you will be able to:</strong>
+<ul>
+   <li>Create and edit a page in Evidence</li>
+   <li>Write a query on demo data</li>
+   <li>Create a table</li>
+   <li>Create a chart</li>
+   <li>Connect a new CSV data source</li>
+</ul>
+</div>
+
+
+<Alert status="info">
+Casey note: I like how the Rails tutorial calls out what you'll achieve, and adds these satisfying green checkmarks. I think it makes the reader feel good and excited. Baked my own CSS for this, happy to change to your suggestions
+</Alert>
+
+## Prerequisites
+
+Please ensure that you have already installed Evidence: [Install Evidence](/install-evidence).
+
+New to web development? Start with the [Basics](#basics).
+
+Familiar with running a server at `localhost`, and writing pages in Markdown? Skip to [Working with data](#working-with-data).
+
+# Basics
+## 1. Start the dev server at localhost:3000
+
+Start your development server, if isn't already running: 
+
+- VSCode: Click `Start Evidence` in the bottom status bar.
```

Comment:
Sometimes it's nice to have Tabs components for things like this so the user can pick which way they want to do something throughout

[VSCode] | [Command Line]

Reply:
I like this but it seems to be having trouble rendering the backticks inside the tabs correctly:

<img width="796" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/0e073586-e86a-433e-8c5f-4aa76ea1314e">

Is there something I need to do to escape these

```
Start your development server, if isn't already running: 

<div class="tab-bottom-padding">
<Tabs>
   <Tab label="VSCode">
         Click `Start Evidence` in the bottom status bar.
   </Tab>
   <Tab label="Command Line">
         Run `npm run dev` in your project directory.
   </Tab>
</Tabs>
</div>

If you chose a different setup during [Install Evidence](/install-evidence), use the command that matches your setup.
```

Reply:
<img width="942" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/4b3a02bc-4c48-4954-a865-c3fcccd50ddc">

guh, still having issues. tell me if I'm doing something dumb